react/features/chat/components/native/styles.js

```diff
@@ -49,6 +50,22 @@ export default {
         fontSize: 13
     },
 
+    inputBar: {
+        borderTopColor: 'rgb(209, 219, 231)',
+        borderTopWidth: 1,
+        flexDirection: 'row',
+        paddingHorizontal: BoxModel.padding
+    },
+
+    inputField: {
+        flex: 1,
+        height: 48
```

Comment:
is this related to MD? if so, a constant?

Reply:
No, this is from Zeplin. I don't thing "they" use MD at all